docs/azure-simple/README.md

```diff
@@ -4,80 +4,77 @@ This document walks through a Bedrock deployment.  It does not include everythin
 
-![Set up empty Flux repository](./images/Win_Subsys_Linux.png)
-
-Per Step 1. in [Prerequisites](#prerequisites) and [the Bedrock documentation](https://github.com/microsoft/bedrock/tree/master/cluster#required-tools), one must install `helm`, `terraform` and `kubectl`.  There are links for each in the documentation.
-
-There is also a set of [scripts](https://github.com/jmspring/bedrock-dev-env/tree/master/scripts) that will install `helm`, `terraform` and `kubectl`.  In this case, use `setup_kubernetes_tools.sh` and `setup_terraform.sh`.  The scripts, which were used for this document, install the tools into `/usr/local/bin`.  
-
-### Install the Azure CLI
+## Install the Azure CLI
 
 For information specific to your operating system, see the [Azure CLI install guide](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest).  You can also use [this script](https://github.com/jmspring/bedrock-dev-env/blob/master/scripts/setup_azure_cli.sh) if running on a Unix based machine.
 
-### Set Up Flux Manifest Repository
+# Set Up Flux Manifest Repository
 
-Create an empty git repository.  We will deploy the Bedrock environment using the empty repo and then add a Kubernetes manifest that defines a simple Web application.  The change to the repo will automatically update the deployment.
+ We will deploy the Bedrock environment using the empty repo and then add a Kubernetes manifest that defines a simple Web application.  The change to the repo will automatically update the deployment.
 
-![Set up empty Flux repository](./images/empty_Repo.png)
+To prepare the Flux manifest repository, we must:
 
-To use the [Flux repository](https://github.com/andrebriggs/sample_app_manifests/tree/master/prod), we must:
+1. [Create the Flux Manifest Repository](#create-the-flux-manifest-repository)
+2. [Generate an RSA Key Pair to use as the Manifest Repository Deploy Key](#generate-an-rsa-key-pair-to-use-as-the-manifest-repository-deploy-key)
+3. [Grant Deploy Key access to the Manifest Repository](#grant-deploy-key-access-to-the-manifest-repository)
 
-1. In order to intialize this repo, do an empty commit, for example: `git commit --allow-empty -m "Hello"`.
-2. Create an RSA keypair for the Flux repository
-3. Add the keypair to the repository
+## Create the Flux Manifest Repository
+[Create an empty git repository](https://github.com/new/) with a name that clearly signals that the repo is used for the Flux manifests. For example `sample_app_manifests`.  
 
-#### Create an RSA Key Pair for a Deploy Key for the Flux Repository
+Do an empty commit, to initialize the repo
```

Comment:
Consider saying "Initialize the repo with an empty commit:" here.

Reply:
done.  added some context of why this is required.